src/components/Insights/Issues/IssuesFilter/index.tsx

```diff
@@ -120,22 +184,48 @@ export const IssuesFilter = ({ query, onApply }: IssuesFilterProps) => {
           <s.FilterCategoryName>Issues</s.FilterCategoryName>
           <Select
             key={"issues"}
-            items={issuesTypeFilter}
-            onChange={handleSelectionChange}
-            placeholder={selectedIssueTypes.length > 0 ? "Issues" : "All"}
+            items={issueTypesFilterOptions}
+            onChange={handleIssueTypesChange}
+            placeholder={filteredInsightTypes.length > 0 ? "Issues" : "All"}
             multiselect={true}
             icon={(props: IconProps) => (
               <s.InsightIconContainer>
-                <InsightsIcon {...props} size={12} color="currentColor" />
+                <FourPointedStarIcon {...props} />
+              </s.InsightIconContainer>
+            )}
+            disabled={issueTypesFilterOptions?.length === 0}
+          />
+          <s.FilterCategoryName>Criticality</s.FilterCategoryName>
+          <Select
```

Comment:
Why  do we need Select for Each if the options ?

Reply:
Filters UI implemented according to the new design: https://www.figma.com/design/MxA95ZHK5NqGX4ptFQljRJ/%E2%AD%90-Design-System-3.0?node-id=9262-121138&m=dev

I suppose there will be more options later on